registrations/tests/test_signup_put.py

```diff
@@ -92,54 +115,102 @@ def test__created_regular_user_can_update_signup(
     user.get_default_organization().regular_users.add(user)
     user.get_default_organization().admin_users.remove(user)
 
-    new_signup_name = "Edited name"
-
     db_signup = SignUp.objects.get(pk=signup.id)
     assert db_signup.first_name != new_signup_name
     assert db_signup.last_modified_by_id is None
 
     signup_data = {
         "registration": registration.id,
         "first_name": new_signup_name,
-        "date_of_birth": "2015-01-01",
+        "date_of_birth": new_date_of_birth,
     }
 
     assert_update_signup(user_api_client, signup.id, signup_data)
-    db_signup = SignUp.objects.get(pk=signup.id)
+
+    db_signup.refresh_from_db()
     assert db_signup.first_name == new_signup_name
     assert db_signup.last_modified_by_id == user.id
 
 
 @freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
-def test__non_created_regular_user_cannot_update_signup(
+def test_non_created_regular_user_cannot_update_signup(
     user_api_client, registration, signup, user
 ):
     user.get_default_organization().regular_users.add(user)
     user.get_default_organization().admin_users.remove(user)
 
-    new_signup_name = "Edited name"
-
     db_signup = SignUp.objects.get(pk=signup.id)
     assert db_signup.first_name != new_signup_name
     assert db_signup.last_modified_by_id is None
 
     signup_data = {
         "registration": registration.id,
         "first_name": new_signup_name,
-        "date_of_birth": "2015-01-01",
+        "date_of_birth": new_date_of_birth,
     }
 
     response = update_signup(user_api_client, signup.id, signup_data)
     assert response.status_code == status.HTTP_403_FORBIDDEN
+    db_signup.refresh_from_db()
+    assert db_signup.first_name != new_signup_name
+    assert db_signup.last_modified_by_id is None
+
+
+@freeze_time("2023-03-14 03:30:00+02:00")
+@pytest.mark.django_db
+def test_created_user_without_organization_can_update_signup(
+    user_api_client, registration, signup, user
+):
+    signup.created_by = user
```

Comment:
Perhaps use factories for both the SignUp and the User.

Reply:
Fixed in https://github.com/City-of-Helsinki/linkedevents/pull/719/commits/4e2de2a39a5ba84b4dd8b8eb1375dc453555fb3b